packages/suite/src/support/messages.ts

```diff
@@ -513,10 +521,26 @@ export default defineMessages({
         defaultMessage: 'Converting',
         id: 'TR_EXCHANGE_DETAIL_CONVERTING_TITLE',
     },
+    TR_EXCHANGE_DETAIL_PROCESSING: {
+        defaultMessage: '{providerName} is processing your swap',
+        id: 'TR_EXCHANGE_DETAIL_PROCESSING',
+    },
     TR_EXCHANGE_DETAIL_CONVERTING_SUPPORT: {
         defaultMessage: 'Go to provider support',
         id: 'TR_EXCHANGE_DETAIL_CONVERTING_SUPPORT',
     },
+    TR_EXCHANGE_HEADING: {
+        defaultMessage: 'Your swap',
+        id: 'TR_EXCHANGE_HEADING',
+    },
+    TR_EXCHANGE_HEADING_DESCRIPTION: {
+        defaultMessage: 'View and track your swap in trade history.',
```

Comment:
```suggestion
        defaultMessage: 'View and track your {type} in trade history.',
```
for buy, sell ,swap

Check `TR_TRADING_DISABLED_DEFAULT` and it's usage. 

```
const typeLabels: Record<TradingDisabledProps['type'], ExtendedMessageDescriptor['id']> = {
    buy: 'TR_BUY',
    sell: 'TR_TRADING_SELL',
    exchange: 'TR_TRADING_SWAP',
};
```